diff --git a/src/args_parser.c b/src/args_parser.c
index 0e055572ed..687d8f5de4 100644
--- a/src/args_parser.c
+++ b/src/args_parser.c
@@ -451,7 +451,7 @@
 
     const char *window = args_get_value_by_key(self->args, NUM_ARGS, "-w");
     snprintf(self->window, sizeof(self->window), "%s", window);
-    self->verbose = args_get_boolean_by_key(self->args, NUM_ARGS, "-v", true);
+    self->verbose = args_get_boolean_by_key(self->args, NUM_ARGS, "-v", false);
     self->gl_debug = args_get_boolean_by_key(self->args, NUM_ARGS, "-gl-debug", false);
     self->record_cursor = args_get_boolean_by_key(self->args, NUM_ARGS, "-cursor", true);
     self->date_folders = args_get_boolean_by_key(self->args, NUM_ARGS, "-df", false);
